src/components/AddTodoForm.jsx

```diff
@@ -9,23 +11,26 @@ const AddTodoForm = ({ onAddTodo }) => {
 
   const handleAddTodo = (event) => {
     event.preventDefault();
-
     onAddTodo(todoTitle);
     setTodoTitle("");
   };
 
   return (
-    <form className="add-todo-form" onSubmit={handleAddTodo}>
-      <input
-        type="text"
-        value={todoTitle}
-        onChange={handleTitleChange}
-        placeholder="Enter a new todo"
-      />
+    <form onSubmit={handleAddTodo}>
+      <InputWithLabel
+        todoTitle={todoTitle}
+        handleTitleChange={handleTitleChange}
+      >
+        Add toDO!
+      </InputWithLabel>
       <button type="submit">Add</button>
     </form>
   );
 };
 
+AddTodoForm.PropTypes = {
```

Comment:
It should be `AddTodoForm.propTypes` instead of `.PropTypes`.
See package documentation: https://www.npmjs.com/package/prop-types

Make the correction in other places.